docs/api.rst

```diff
@@ -0,0 +1,35 @@
+API Documentatie
+================
+
+De PABC applicatie biedt een REST API voor integratie met externe systemen.
+
+OpenAPI Specificatie
+--------------------
+
+De API is gedocumenteerd met behulp van OpenAPI 3.0 specificatie.
+
+Toegang tot de API Documentatie
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**JSON Specificatie:**
+
+De volledige OpenAPI specificatie is beschikbaar op:
+
+.. code-block:: 
+
+   GET /swagger/v1/swagger.json
+
+**Voorbeelden:**
+
+* Development: ``http://localhost:5211/swagger/v1/swagger.json``
```

Comment:
even wachten op een mogelijke aanpassing in https://github.com/PodiumD-Autorisatie-Beheer-Component/PABC-API/pull/24